### ISSUE TITLE
Lost mined $IRONS

### ISSUE THREAD

**Issue by sunsweet733** [iron-fish/ironfish#1374]
### What happened?

After upgrading from 0.13.2 to 0.13.3 and using the ironfish accounts:rescan --reset command, my previously mined $IRONS disappeared

Screenshot of mined IRONS: http://joxi.ru/p27BK7ptZvWzDm

Now it looks like this: http://joxi.ru/Q2K9l7kCg3Me4r

### Version

v 0.13.2 -> 0.13.3

### Debug output

```shell
yarn run v1.22.18
$ yarn build && yarn start:js debug
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node bin/run debug
Database in use, skipping output that requires database.
Stop the node and run the debug command again to show full output.

Iron Fish version       0.1.33 @ src
Iron Fish library       0.0.11 @ src
Operating system        Linux x64
CPU model(s)            Common KVM processor
CPU threads             40
RAM total               11.41 GiB
Node version            v16.15.0
ironfish in PATH        true
Telemetry enabled       true
Done in 4.94s.
```

### Graffiti

ssw73

**Comment by wd021** [comment]
duplicate of https://github.com/iron-fish/ironfish/issues/1340

**Comment by lwisne** [comment]
did your tokens ever come back here?

**Comment by lwisne** [comment]
attacking files directly: 
![1652456229011](https://user-images.githubusercontent.com/27844/182728086-ea0196a4-4d82-4405-a321-0ceded5bd2d1.png)
![1652356389560](https://user-images.githubusercontent.com/27844/182728091-42004995-66dd-43d4-b28c-4249fc5f5b08.png)

**Comment by lwisne** [comment]
It seems likely the original larger balance was in error due to another bug.

I see `ssw73` having mined 3 blocks in the explorer. This would give you a little more than 60 $IRON. 

Having 118 $IRON would be unlikely having only mined these 3 blocks.

Any reason to believe you should have legitimately had that larger amount?

**Comment by hughy** [comment]
The Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48) included significant changes to the way that account balances are stored and calculated.

Please open a new issue if you run into this again!
